### ISSUE TITLE
Move password for slide upload to configuration file

### ISSUE THREAD

**Issue by shoeffner** [jspricke/ik-infoscreen#2]
Currently the password for the slide upload is publicly exposed (in this repository). We should change it and move it to a configuration file.

**Comment by jspricke** [comment]
I think it's enough to change it on the on site server. But thanks for 
the reminder :).

**Comment by jspricke** [comment]
I would propose to drop the password. We do a review of the incoming data anyhow.

**Comment by v4hn** [comment]
Agreed. See #23.

The reasoning for it back then was to keep away arbitrarily many big anonymous uploads.
I would propose to set a maximum quota for uploads instead.

**Comment by shoeffner** [comment]
The mixed reality session had some very large files, right?

We can remove the password without many problems, and maybe upload to different directory so we have some kind of a staging/moderation area, if that is what you have in mind?

**Comment by jspricke** [comment]
Uploads go to /incoming/files which is non public already, so we can review it. Also, `php.ini` restricts the upload size. We currently set it to: `post_max_size = 500M`.

**Comment by v4hn** [comment]
Yes, uploads have to be moved to a public location by hand already and *individual* file size is restricted too.
I was more concerned about cumulative storage space and would prefer to have a quota on the maximum overall size of unreviewed uploaded files.
